client/src/components/SandwichShow.js

```diff
@@ -98,10 +99,53 @@ const SandwichShow = (props) => {
     );
   });
 
+  const roundHalf = (num) => {
+    return Math.round(num*2)/2
+  }
+
```

Comment:
this roundHalf function could also be moved into service function I mention below